spectacles/validators.py

```diff
@@ -311,19 +313,17 @@ def _create_queries(self, mode: str) -> List[Query]:
     def _create_explore_query(self, explore: Explore, model_name: str) -> Query:
         """Creates a single query with all dimensions of an explore"""
         dimensions = [dimension.name for dimension in explore.dimensions]
-        query_id = self.client.create_query(model_name, explore.name, dimensions)
-        return Query(query_id, lookml_ref=explore)
+        query = self.client.create_query(model_name, explore.name, dimensions)
+        return Query(query["id"], lookml_ref=explore, query_url=query["share_url"])
```

Comment:
Since we're returning the full response from the API now, could you add an `@classmethod` alternate constructor to the Query class called `from_json` that accepts the full response and builds a query? This is similar to the LookML objects. Nice and clean and you don't have to pass args really you can just pass that `query` variable directly like this:
```
return Query.from_json(query)
```

Reply:
We would still need to pass the `lookml_ref` manually, right? It's not part of the JSON response.

Reply:
Oh you're right. I guess it's a question of is
```
return Query.from_json(query, lookml_ref=explore)
```
Really that much better. It's your call, I might be being overly picky here.

Reply:
For the moment, I think I'd rather leave as is. It feels like overkill if we only pass two arguments, though I think if we add any more I agree we should add the functionality.